.github/workflows/release.yml

```diff
@@ -58,6 +59,13 @@ jobs:
           annotations: ${{ steps.metadata.outputs.annotations }}
           labels: ${{ steps.metadata.outputs.labels }}
           tags: ${{ steps.metadata.outputs.tags }}
+
+  docker-hub-description:
+    needs: docker-hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
The previous job checked out the upstream code, so the upstream code's README is the one that's uploaded. Now the right README will be uploaded ...

<img width="1825" alt="Screenshot 2024-11-19 at 14 18 10" src="https://github.com/user-attachments/assets/c3f9ecb1-ad65-430d-9e7f-6dd7fcc56f45">

Reply:
Since this won't run on merge (only on the next tag), I've manually updated the `README` on Docker Hub ...

<img width="1825" alt="Screenshot 2024-11-19 at 14 34 35" src="https://github.com/user-attachments/assets/e8ece056-44a2-4780-9086-9b5feba62092">